Comanche055/TVCINITIALIZE.agc

```diff
@@ -368,21 +368,21 @@ CSMN10		DEC	.99999		# N10	CSM ONLY FILTER COEFFICIENTS
 
 LBN10		DEC	+.99999		# N10	LOW BANDWIDTH FILTER COEFFICIENTS
 		DEC	-.3285		# N11/2
-		DEC	-.3301		#N12
-		DEC	-.9101		#D11/2
-		DEC	+.8460		#D12
-
-		DEC	+.03125		#N20
-		DEC	0		#N21/2
-		DEC	0		#N22
-		DEC	-.9101		#D21/2
-		DEC	+.8460		#D22
-
-		DEC	+.50000		#N30
-		DEC	-.47115		#N31/2
-		DEC	+.4749		#N32
-		DEC	-.9558		#D31/2
-		DEC	+.9372		#D32
+		DEC	-.3301		# N12
```

Comment:
Thanks for fixing the inconsistency. Actually though, hashtags shouldnt be used for comments at all since theyre not in the original scans. I believe they were added so the source would compile for the VirtualAGC project. The right thing to do would be to remove them all, but you can do that in another PR if youre up to it. Thanks!